src/System.Private.CoreLib/src/System/RuntimeTypeHandle.cs

```diff
@@ -89,17 +89,6 @@ public bool Equals(RuntimeTypeHandle handle)
             return true;
         }
 
-        public IntPtr Value => _value;
-
-        public ModuleHandle GetModuleHandle()
```

Comment:
I find this API offensive too, but this is a [public API](https://docs.microsoft.com/en-us/dotnet/api/system.runtimetypehandle.getmodulehandle?view=netcore-3.1)

Reply:
oh oops 🤣